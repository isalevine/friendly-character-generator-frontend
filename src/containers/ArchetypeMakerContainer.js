import React, {Component} from 'react';

import ArchetypeMakerForm from '../components/forms/ArchetypeMakerForm'


class ArchetypeMakerContainer extends Component {

    render() {
        return (
            <div id="archetype-maker-container">
                <ArchetypeMakerForm />
            </div>
        )
    }

}

export default ArchetypeMakerContainer