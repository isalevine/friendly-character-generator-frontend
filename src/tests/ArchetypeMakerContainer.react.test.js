import React from 'react';
import renderer from 'react-test-renderer';
import ArchetypeMakerContainer from '../containers/ArchetypeMakerContainer';

test ('ArchetypeMakerContainer is a renderable component', () => {
    const component = renderer.create(
        <ArchetypeMakerContainer />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})

