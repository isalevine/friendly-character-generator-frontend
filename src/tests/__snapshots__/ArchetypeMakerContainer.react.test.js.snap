// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArchetypeMakerContainer is a renderable component 1`] = `
<h1>
  hi
</h1>
`;
